matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/room/send/EncryptEventWorker.kt

```diff
@@ -80,7 +78,7 @@ internal class EncryptEventWorker(context: Context, params: WorkerParameters)
         var result: MXEncryptEventContentResult? = null
         try {
             result = awaitCallback {
-                crypto.encryptEventContent(localMutableContent, localEvent.type, params.roomId, it)
+                crypto.encryptEventContent(localMutableContent, localEvent.type, localEvent.roomId!!, it)
```

Comment:
we are pretty sure it will never be null. It is nullable because Event data class is also use to parse what come form the server, and we do not trust what the server sends.  But in this case, we have built the local echo and have set a roomId